contracts/src/token/erc721/extensions/consecutive.rs

```diff
@@ -11,20 +11,18 @@
 //! contract construction. This ability is regained after construction. During
 //! construction, only batch minting is allowed.
 //!
-//! IMPORTANT: Function [`Erc721Consecutive::_mint_consecutive`] is not suitable
-//! to be called from constructor. Because of stylus sdk limitation. Function
-//! [`Erc721Consecutive::_stop_mint_consecutive`] should be called to end
-//! consecutive mint of tokens. After that, minting a token
-//! with [`Erc721Consecutive::_mint`] will be possible.
+//! Fields `_first_consecutive_id` (used to offset first token id) and
+//! `_max_batch_size` (used to restrict maximum batch size) can be assigned
+//! during construction with `koba` (stylus construction tooling) within
+//! solidity constructor file.
 //!
-//! IMPORTANT: When overriding [`Erc721Consecutive::_mint_consecutive`], be
-//! careful about call ordering. [`Erc721Consecutive::owner_of`] may return
-//! invalid values during the [`Erc721Consecutive::_mint_consecutive`]
-//! execution if the super call is not called first. To be safe, execute the
-//! super call before your custom logic.
+//! IMPORTANT: Consecutive mint of [`Erc721Consecutive`] tokens is only allowed
+//! inside solidity constructor with koba.
+//! Compared to solidity implementation of consecutive contract, there is no
+//! restriction for [`Erc721Consecutive::_update`] function call. Since it won't
+//! be possible to call a rust function from solidity constructor.
```

Comment:
```suggestion
//! IMPORTANT: Consecutive mint of [`Erc721Consecutive`] tokens is only allowed
//! inside the contract's Solidity constructor.
//! As opposed to the Solidity implementation of Consecutive, there is no
//! restriction on the [`Erc721Consecutive::_update`] function call since it is not
//! possible to call a Rust function from the Solidity constructor.
```